app/models/endpoint.rb

```diff
@@ -41,6 +41,18 @@ def ssl_cert_store
     nil # use system defaults
   end
 
+  # Split concatenated PEM certs to a list
+  def certificate_authorities
+    return [] if certificate_authority.blank?
+    certificate_authority.split(/(?=-----BEGIN)/).reject(&:blank?)
```

Comment:
PEM allows only alphanumerics, `+`, `/` and padding `=`.  That's why it can
use dashes as unambiguous boundarries.
https://en.wikipedia.org/wiki/Base64#Privacy-enhanced_mail

But great point, will do.